code/lib/cli/src/automigrate/fixes/remove-jest-testing-library.ts

```diff
@@ -0,0 +1,32 @@
+import chalk from 'chalk';
+import dedent from 'ts-dedent';
+import type { Fix } from '../types';
+
+export const removeJestTestingLibrary: Fix<{ incompatiblePackages: string[] }> = {
+  id: 'remove-jest-testing-library',
+  promptOnly: true,
+  async check({ mainConfig, packageManager }) {
+    const deps = await packageManager.getAllDependencies();
+
+    const incompatiblePackages = Object.keys(deps).filter(
+      (it) => it === '@storybook/jest' || it === '@storybook/testing-library'
+    );
+    return incompatiblePackages.length ? { incompatiblePackages } : null;
+  },
+  prompt({ incompatiblePackages }) {
+    return dedent`
+      ${chalk.bold(
+        'Attention'
+      )}: We've detected that you're using the following packages which are known to be incompatible with Storybook 8:
+
+      ${incompatiblePackages.map((name) => `- ${chalk.cyan(`${name}`)}`).join('\n')}
+      
+      Install the replacement for those packages: ${chalk.cyan('@storybook/test')}
+      
+      And run the following codemod:
+       ${chalk.cyan(
+         'npx sb migrate migrate-to-test-package --glob="**/*.stories.@(js|jsx|ts|tsx)"'
```

Comment:
As far as I know we shouldn't recommend `sb`, but `storybook` instead.

```suggestion
         'npx storybook@latest migrate migrate-to-test-package --glob="**/*.stories.@(js|jsx|ts|tsx)"'
```

Reply:
Discussed with @shilman, but in this case it is better to use the local `storybook` binary, as if we would delete this codemod in feature version, it won't be there. But the local codemod will always be there.